### ISSUE TITLE
No more view count

### ISSUE THREAD

**Issue by FunixG** [FunixG/TwitchLib#4]
DEATH TO VIEW COUNTS!

A change is coming:

As of today, messaging has been added to the Twitch Stream Manager indicating that the “views” count along the top of the page will be removed on April 14, 2022. For further information regarding how this change affects broadcasters, please see the Stream Stats section of the Stream Manager Twitch Help Article.

https://discuss.dev.twitch.tv/t/get-users-api-endpoint-view-count-deprecation/37777

see also https://help.twitch.tv/s/article/stream-manager?language=en_US#stats

TLDR: the "total views" a User has, is being completely removed from the Platform.

Beginning April 15, 2022, the data in “view_count” will not be updated and will contain stale data.
At a later time, we will remove the “view_count” field from the Get Users response payload entirely.
